test/stores/notifications/RoomNotificationState-test.ts

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Room } from "matrix-js-sdk";
+
+import "../../skinned-sdk";
+import { stubClient } from "../../test-utils";
+import { MatrixClientPeg } from "../../../src/MatrixClientPeg";
+import { RoomNotificationState } from "../../../src/stores/notifications/RoomNotificationState";
+import * as testUtils from "../../test-utils";
+import { NotificationStateEvents } from "../../../src/stores/notifications/NotificationState";
+
+describe("RoomNotificationState", () => {
+    stubClient();
+    const client = MatrixClientPeg.get();
```

Comment:
we should probably come up with a code style for tests - these should be in a `beforeEach` (or `beforeAll` depending on your scope), but we don't really have a consistent approach within our existing tests.

Will bring it up at another venue.

Reply:
Yeah, it seemed a bit presumptuous to assume that all the tests would want something when it was the only test in the file so far... probably exactly why we should have consistent best practices.